dockerfiles/cli/scripts/cli.sh

```diff
@@ -111,11 +111,52 @@ cmd_init_reinit_pre_action() {
   fi
 }
 
+check_all_ports(){
+  docker run -d -p 80:80 \
+                -p 443:443 \
+                -p 2181:2181 \
+                -p 5000:5000 \
+                -p 23750:23750 \
+                -p 23751:23751 \
+                -p 32001:32001 \
+                -p 32101:32101 \
+                --name fake ${UTILITY_IMAGE_ALPINE} \
+                                 httpd -f -p 80 \
+                                          -p 443 \
```

Comment:
I did a small test with it and without it because I thought the same thing.  However, without the httpd, the test would pass even though the port was bound.